frontend/src/metabase/search/components/SearchResult/components/Context.styled.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
+
+export const ContextText = styled("p")`
```

Comment:
`ContextText` & `ContextContainer` should be removed from `SearchResult.styled.tsx`